test/plots/stackedAreaPlotTests.ts

```diff
@@ -2,485 +2,346 @@
 
 describe("Plots", () => {
   describe("Stacked Area Plot", () => {
-    let svg: d3.Selection<void>;
-    let dataset1: Plottable.Dataset;
-    let dataset2: Plottable.Dataset;
-    let xScale: Plottable.Scales.Linear;
-    let yScale: Plottable.Scales.Linear;
-    let renderer: Plottable.Plots.StackedArea<number>;
-    let SVG_WIDTH = 600;
-    let SVG_HEIGHT = 400;
-
-    beforeEach(() => {
-      svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      xScale = new Plottable.Scales.Linear();
-      xScale.domain([1, 3]);
-      yScale = new Plottable.Scales.Linear();
-      yScale.domain([0, 4]);
-      let colorScale = new Plottable.Scales.Color("10").domain(["a", "b"]);
-
-      let data1 = [
-        {x: 1, y: 1, type: "a"},
-        {x: 3, y: 2, type: "a"}
-      ];
-      let data2 = [
-        {x: 1, y: 3, type: "b"},
-        {x: 3, y: 1, type: "b"}
-      ];
-      dataset1 = new Plottable.Dataset(data1);
-      dataset2 = new Plottable.Dataset(data2);
-
-      renderer = new Plottable.Plots.StackedArea<number>();
-      renderer.addDataset(dataset1);
-      renderer.addDataset(dataset2);
-      renderer.x((d) => d.x, xScale);
-      renderer.y((d) => d.y, yScale);
-      renderer.attr("fill", "type", colorScale);
-      let xAxis = new Plottable.Axes.Numeric(xScale, "bottom");
-      new Plottable.Components.Table([[renderer], [xAxis]]).renderTo(svg);
+    describe("setting the x property", () => {
+      let svg: d3.Selection<void>;
+      let stackedAreaPlot: Plottable.Plots.StackedArea<number>;
+
+      beforeEach(() => {
+        svg = TestMethods.generateSVG();
+        let data1 = [
+          {foo: 1},
```

Comment:
from the usage, `foo` could be `x` and `bar` could be `offset`, but then again, it's just personal preference as there's not a clear mapping